test/unittests/utils/test_pet.py

```diff
@@ -0,0 +1,115 @@
+import pandas as pd
+import pytest
+
+
+@pytest.fixture
+def psf_df():
+    return pd.DataFrame(
+        {
+            "participant_id": ["sub-CLNC01"] * 3 + ["sub-CLNC02", "sub-CLNC03"],
+            "session_id": ["ses-M00", "ses-M18"] + ["ses-M00"] * 3,
+            "acq_label": ["FDG", "FDG", "AV45", "FDG", "FDG"],
+            "psf_x": [8, 8, 7, 8, 8],
+            "psf_y": [9, 9, 6, 9, 9],
+            "psf_z": [10, 10, 5, 10, 10],
+        }
+    )
+
+
+def test_read_psf_information_errors(tmp_path, psf_df):
+    from clinica.utils.pet import read_psf_information
+
+    with pytest.raises(
+        FileNotFoundError,
+        match="No such file or directory: 'foo.tsv'",
+    ):
+        read_psf_information(
+            "foo.tsv", ["sub-CLNC01", "sub-CLNC01"], ["ses-M00", "ses-M18"], "FDG"
```

Comment:
"Expected type 'PathLike', got 'str' instead"

Not a big deal, but if we don't want type alignment warnings, it should be wrapped in `pathlib.PurePath`.